src/TestSuite/Fixture/FixtureLoader.php

```diff
@@ -79,17 +79,26 @@ abstract public function loadSingle(
     abstract public function load(TestCase $test): void;
 
     /**
-     * Inspects the test to to load fixture classes.
+     * Setup fixtures for the provided test.
      *
-     * @param \Cake\TestSuite\TestCase $test The test case to inspect.
+     * Called by TestCase during its setUp() method.
+     *
+     * @param \Cake\TestSuite\TestCase $test The test to inspect for fixture loading.
      * @return void
      */
-    abstract public function fixturize(TestCase $test): void;
+    abstract public function setupTest(TestCase $test): void;
 
     /**
-     * Get the fixtures fixtures.
+     * Get the list of all fixtures that have been loaded.
      *
      * @return \Cake\Datasource\FixtureInterface[]
      */
     abstract public function loaded(): array;
+
+    /**
+     * Get the list of fixture tables that were loaded since the last call to setupTest.
+     *
+     * @return string[]
+     */
+    abstract public function lastInserted(): array;
```

Comment:
This name doesn't make much sense. Fixtures aren't "inserted" and "last" doesn't really mean anything either.

I know this was a hack/temp solution but maybe clarifying this will help.

Reply:
I agree the name is clunky. I am planning to make this method protected when I move state management. I didn't want to increase the scope of this change too much as I am still battling with the various database differences.